packages/concerto-core/test/decoratormanager.js

```diff
@@ -188,6 +188,21 @@ describe('DecoratorManager', () => {
             decoratorCity2Property.should.not.be.null;
         });
 
+        it('should decorate the specified element on the specified Map Declaration (Map Declaration)', async function() {
```

Comment:
LGTM, but I would change this to reflect the test accurately, a MapDeclaration is not an element on a MapDeclaration.

```suggestion
        it('should decorate the specified MapDeclaration', async function() {
```

Reply:
Addressed comment suggestion.